internal/host/host.go

```diff
@@ -183,6 +185,28 @@ func (m *Manager) UpdateInventory(ctx context.Context, h *models.Host, inventory
 	return m.db.Model(h).Update("inventory", inventory).Error
 }
 
+func (m *Manager) UpdateTimestamp(ctx context.Context, h *models.Host, timestamp int64) error {
```

Comment:
When we spoke about having the timestamp as part of inventory, I assumed the agent would be sending it together with the inventory.  If you're sending it separately, which is fine, I would put timestamp as a separate property.  What do you think?

Reply:
Agree.

Reply:
we will send it as part of inventory. 
i just need an option to send it without inventory too. 
As i wrote in description we will get it in 2 ways cause we are not running inventory check on Known state but we need to get timestamp.

Reply:
we can set as new host property. though it will be less readable to have it as some  new strange host field

Reply:
moving all to inventory